common/components/common/projects/EditProjectForm.jsx

```diff
@@ -318,13 +324,37 @@ class EditProjectForm extends React.PureComponent<Props,State> {
             }
           </label>
           <div className="character-count">
-            { (this.state.formFields.project_description || "").length} / 3000
+            { (this.state.formFields.project_description || "").length} / 1000
```

Comment:
For this field we're going to want to keep the existing 3000 character limit in order to accommodate projects who already have long descriptions.